.travis.yml

```diff
@@ -1,7 +1,10 @@
 language: python
 python:
-    - "2.6"
     - "2.7"
+    - "3.2"
+    - "3.3"
+    - "3.4"
+    - "3.5"
 install:
-    - pip install -r requirements.txt --use-mirrors
+    - pip install virtualenv==13.1.2
```

Comment:
Hm. I suspect that something isn't right if specifying this is necessary. I have projects with the same python dependencies where I don't need to do this. One difference though is that I'm explicitly using their contained based infrastructure. Can you add `sudo: false` to the top of this file and remove the install section just to see if there is a difference?